README.md

```diff
@@ -121,7 +123,7 @@ npm start
 
 ## 🔭 Future Features <a name="future-features"></a>
 
```

Comment:
- [ ] The issues link is broken kindly make sure to fix it so all the likes in your README are working properly